docs/schemas/actions.yml

```diff
@@ -33,6 +33,9 @@ components:
           type: string
           description: Reference to a parent transaction (e.g., token lock reference for token unlocks or allow spend reference for spend transactions)
           nullable: true
+        snapshotOrdinal:
```

Comment:
Is this global or metagraph snapshot ordinal? I think we should be explicit here to avoid confusion.